R/load_files.R

```diff
@@ -11,7 +11,7 @@
 #' 
 #' @examples
 #'  td <- get_test_data()
-#'  l_tbl <- load_data(td$m_file, td$t_files, td$r_files)
+#'  l_tbl <- load_data(manifest_path(td), template_path(td), result_path(td))
```

Comment:
Now that we have the S4 object, we can use it by taking the object itself in as the argument like `load_data(td)`. Then, the `manifest_path`, `template_path`, `result_path`, etc. will be taken from the object

Reply:
:)